src/indieweb_utils/replies/context.py

```diff
@@ -374,9 +374,12 @@ def get_reply_context(url: str, twitter_bearer_token: str = "", summary_word_lim
     if page_content.status_code != 200:
         raise ReplyContextRetrievalError(f"Page returned a {page_content.status_code} response.")
 
-    webmention_endpoint_url_response = discover_webmention_endpoint(url)
+    try:
+        webmention_endpoint_url_response = discover_webmention_endpoint(url)
 
-    webmention_endpoint_url = webmention_endpoint_url_response.endpoint
+        webmention_endpoint_url = webmention_endpoint_url_response.endpoint
+    except Exception:
```

Comment:
Can we catch a more specific `Exception`? i.e. only the errors we're expecting? Or does that not make sense in this function?

Reply:
There are three different exceptions that could be raised here. Should I add them in or does the more generic Exception make sense? I understand having an explicit exception is best but the outcome for any of the three raised would be the same: set the endpoint URL to a blank string.

Reply:
I'd catch the specific errors we're expecting and set the endpoint URL to a blank string. That way any other errors/bugs aren't masked by this broad handler.

```python
try:
   ...
except (AException, BException, CException):
   ...
```